### ISSUE TITLE
Windows x64/x86 shared fails to build

### ISSUE THREAD

**Issue by phosmium** [pmed/v8-nuget#32]
Looks like windows x64/x86 fails to build on latest and 11.9.169.4. Tested via github workflow and manual build. Seems like an linker error with abseil.

[build_log.txt](https://github.com/pmed/v8-nuget/files/13949720/build_log.txt)

**Comment by pmed** [comment]
Yes, there was a build issue with abseil. I've updated the build script.
